docs/order-with-promo/DateInput.re

```diff
@@ -0,0 +1,13 @@
+let stringToDate = s =>
+  // add "T00:00" to make sure the date is in local time
+  s ++ "T00:00" |> Js.Date.fromString;
```

Comment:
I think this should be `+00:00` (`+`, not `T`).

In general, I wonder if there are safer ways to handle local times than concatenating strings. Why can't we store the date together with the timezone of the user? 

```
let t = new Date("2020-04-13T00:00:00.000")
t
Mon Apr 13 2020 00:00:00 GMT+0200 (Central European Summer Time)
t.toISOString()
'2020-04-12T22:00:00.000Z'
```

Reply:
Sorry, I got confused.

re: the printing, maybe `toLocaleDateString` could be used? not sure.